### DISCUSSION TITLE
Headlands jsut on the Fields ends with a center field cut between for x lanes

### DISCUSSION THREAD

**Discussion by Darnulf** [Courseplay/Courseplay_FS22#3058] [Ideas]
Headlands just goes in circles around the field- specially on fields with a hardboundary as walls, trees or  fences it sometimes  is tedious when the equippment gets stuck and be more beneficial to just create a headlands of x rows cut from there through the field to the other and and create an equal headland and maybe expand the cutting lane by x rows to make the path wider just basically creating 2 beds then left and right of it but leave off the side headlands done with a roundtrip headland.
Would be a nice to have.

**Comment by Tensuko** [comment]
Is part of the new generator and was called narrow field.
But takes a while until we have that rdy.
